src/shipit_code_coverage/shipit_code_coverage/codecov.py

```diff
@@ -71,6 +74,35 @@ def __init__(self, revision, cache_root, coveralls_token, codecov_token,
 
         logger.info('Mercurial revision', revision=self.revision)
 
+    def get_chunks(self):
+        return list(set([f.split('_')[1] for f in os.listdir('ccov-artifacts')]))
+
+    def get_coverage_artifacts(self, suite=None, chunk=None):
```

Comment:
Most of the code in this function was in `generate_info` and was moved to a separate function for readability.